File: users-microservice/common/stringUtils.js
const generateRandomHexString = (size = 24) => {
  let result = '';
  const characters = 'abcdef0123456789';
  for (let i = 0; i < size; i++) {
    result += characters.charAt(Math.floor(Math.random() * characters.length));
  }
  return result;
}

module.exports = {
  generateRandomHexString,
}

File: books-microservice/src/main/java/com/github/juliocesarscheidt/books/common/StringUtils.java
package com.github.juliocesarscheidt.books.common;

public class StringUtils {

  public static String generateUuid(int size) {
    String result = "";
    String characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
    for (int i = 0; i < size; i++) {
      result += characters.charAt((int) Math.floor(Math.random() * characters.length()));
    }
    return result;
  }
}
